indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -830,3 +830,44 @@ dd > .i-has-action {
         padding: 0.25em 0;
     }
 }
+
+
+.color-palette {
+    background: #FFF;
```

Comment:
do we have any preference for color names vs hex codes? at least for white/black `#fff` and `#000` are kind of obvious